src/common/components/layouts/pages/inspect/Inspect.js

```diff
@@ -96,6 +113,9 @@ function Inspect({ jobStatus, taskStatus, ruleSummaries, lockApp, unlockApp, onF
             setExpandedNodes(initialExpandState);
         }
     }, [treeData, initialExpandState]);
+    useEffect(() => {
+        setExpandedGroups(new Array(errorTags[selectedGroup].length + 1).fill(UNSELECTED).map((_, index) => index));
```

Comment:
_Minor:_ We can skip the step with filling an array with empty values: `Array.from(Array(errorTags[selectedGroup].length + 1).keys())`